modules/ROOT/nav.adoc

```diff
@@ -259,7 +259,7 @@
 * xref:validation/validation-connector.adoc[Validation Module]
  ** xref:validation/validation-documentation.adoc[Validation Connector Reference]
 * xref:vm/vm-connector.adoc[VM Connector]
- ** xref:vm/vm-publish-listen.adoc[Example: Publish and Get a Response in the VM Connector]]
+ ** xref:vm/vm-publish-listen.adoc[Example: Publish and Listen to Messages with the VM Connector]]
```

Comment:
Why is this necessary? Which concerns did you have regarding the old tittle?

Reply:
The old title there twice. So I changed the first occurrence to reflect the example the link to that example in the See Also section.(There is an error in the new ink that I need to correct.) Here is the duplicate occurrence: 
![Screen Shot 2019-08-26 at 9 10 29 AM](https://user-images.githubusercontent.com/46829037/63693205-7110b480-c7e1-11e9-9e43-b37c3debdbc7.png)